src/Features/Core/Portable/InheritanceMargin/AbstractInheritanceMarginService_Helpers.cs

```diff
@@ -247,9 +247,14 @@ internal abstract partial class AbstractInheritanceMarginService
                     foreach (var import in group)
                     {
                         var item = DefinitionItem.Create(
-                            [], [],
-                            new DocumentSpan(destinationDocument, import.DeclaringSyntaxReference!.Span),
-                            classifiedSpans: null);
+                            tags: [],
+                            displayParts: [],
+                            sourceSpans: [new DocumentSpan(destinationDocument, import.DeclaringSyntaxReference!.Span)],
+                            classifiedSpans: [null],
+                            metadataLocations: [],
+                            nameDisplayParts: default,
+                            displayIfNoReferences: true);
```

Comment:
is the `:true` here the same semantics as before?

Reply:
Yes, true has been the default.